### PR TITLE
E1662 - UI Issues/Fixes

### PR DESCRIPTION
**Pull request by psabhyan**
All the changes for the OSS project are done. Following issues fixed:
Issue#295
Issue#316
Issue#702

### CODE REVIEW

File: app/views/sign_up_sheet/_table_header.html.erb [hunk 1]
```diff
@@ -8,9 +8,13 @@
   <th width="5%">Available slots</th>
   <th width="5%">Num. on waitlist</th>
   <th width="10%">Bookmarks</th>
+<%if !@assignment.get_current_stage.eql?"Finished"%>
   <th width="10%">Actions</th>
+<% end %>
+<%if !@assignment.get_current_stage.eql?"Finished"%>
```
Comment (Winbobob) [inline app/views/sign_up_sheet/_table_header.html.erb]: Why do you use 2 `if` block here?

**Comment by yangsong8** [comment]
I have reviewed your pull request:

- issue #295 does not work. I created a review rubric and deleted it, but there was no popup.
- issue #256 was not done.

File: app/views/course/_course.html.erb [hunk 2]
```diff
@@ -8,5 +8,10 @@
 <p><label for="course_info">Course Information</label><br/>
 <%= text_field 'course', 'info'  %></p>
 
+<!--Adding a field so that user can select institution from the list-->
+<p><label for="course_institutions_id">Institution Name</label><br/>
```
Comment (yangsong8) [inline app/views/course/_course.html.erb]: This display works, but nothing got saved at backend.

File: app/models/institution.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 class Institution < ActiveRecord::Base
   validates_length_of :name, minimum: 1
   validates_uniqueness_of :name
+  # Establishing relationship between Course and Institution
+  has_many :courses, dependent: :destroy
```
Comment (yangsong8) [inline app/models/institution.rb]: This means deleting one institution will cause a bunch of course got deleted. Does not sound like the right thing to do.

**Pull request closed without merging**